book/scheduling.md

```diff
@@ -328,6 +308,20 @@ class TaskRunner:
         self.condition.release()
 ```
 
+It's important to call `wait` at the end of the `run` loop if there is nothing
```

Comment:
Not critical IMO

Reply:
I thought this was weirdly placed (it’s a move I think)

Reply:
But not critical.